src/components/TrackTile/TrackerDetails/DatePicker.tsx

```diff
@@ -11,10 +11,6 @@ declare module './TrackerDetails' {
   interface Styles extends StylesProp<typeof defaultStyles> {}
 }
 
-declare module './AdvancedTrackerDetails' {
-  interface Styles extends StylesProp<typeof defaultStyles> {}
-}
```

Comment:
I'm reading through each file and have seen removal of this kind of thing a time or two, but not sure I understand it.  Is creating this module done elsewhere now, or is this more of a general shift to instead be tied to the theme/style provider in the SDK (as evident from the nice simplification in the `TrackTileProvider` file above).

Reply:
I removed it because it was causing type errors and isn't needed.  It is part of the old system of generating types for custom user styles and is being replaced by our updated system in the SDK